app/src/components/BackLink.tsx

```diff
@@ -0,0 +1,17 @@
+import { useTranslation } from 'next-i18next'
+import { ReactElement } from 'react'
+
+import StyledLink from '@components/StyledLink'
+
+export interface BackLinkProps {
+  href: string
+}
+
+export const BackLink = (props: BackLinkProps): ReactElement => {
+  const { href } = props
+
+  const { t } = useTranslation('common')
+  return <StyledLink href={href} text={t('backlinkText')} />
+}
```

Comment:
@rocketnova out of curiosity -- is the main function of this component to translate the back link text before using the styled link?